suites/pacific/upgrades/tier-1_upgrade_cephadm.yaml

```diff
@@ -160,20 +160,39 @@ tests:
       destroy-cluster: false
       abort-on-fail: true
   - test:
-      name: Upgrade ceph
-      desc: Upgrade cluster to latest version
-      module: test_cephadm_upgrade.py
-      polarion-id: CEPH-83573791
-      config:
-        command: start
-        service: upgrade
-        base_cmd_args:
-          verbose: true
-        benchmark:
-          type: rados                      # future-use
-          pool_per_client: true
-          pg_num: 128
-          duration: 10
-        verify_cluster_health: true
-      destroy-cluster: false
-      abort-on-fail: true
+      name: Parallel run
+      module: test_parallel.py
+      parallel:
+        - test:
+            name: rgw sanity tests
+            module: sanity_rgw.py
+            config:
+                script-name: test_multitenant_user_access.py
+                config-file-name: test_multitenant_access.yaml
+                timeout: 300
```

Comment:
Instead of creating user, I would suggest to execute "test_Mbuckets_with_Nobjects.yaml"  as a part parallel run

Ref: https://github.com/red-hat-storage/cephci/blob/00d3799ce21a2ce8e17f1c90cc3b1aba7b149934/suites/pacific/rgw/tier-0_rgw.yaml#L74

Reply:
@udaysk23 http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-YAD52I/Parallel_run_0.log can you please look at the rgw i/o isssue happening here

Reply:
@Ameenasuhani I am looking into this issue

Reply:
@Ameenasuhani By looking at log, I can see that we are not deploying RGWs on cluster.
Due to this scrips is not able to find RGW endpoint and we are hitting this issue.
Could you please look into following logs:
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-678V0D/Cluster_deployment_0.log

Reply:
Thanks, @udaysk23 this resolves the issue